lib/IDE/Refactoring.cpp

```diff
@@ -3147,26 +3147,24 @@ bool RefactoringActionLocalizeString::performChange() {
 }
 
 struct MemberwiseParameter {
-  Identifier Name;
+  CharSourceRange NameRange;
```

Comment:
I wonder if `Identifier` should have a bit for whether it needs backticks or not. I think there's a bunch of bugs involving not adding backticks and it'd be nice if we instead had a method in `Identifier` that would get the *actual* text for the name.

Reply:
I don’t think it should because we use pointer equality to compare `Identifier`s and we want to consider two identifiers identical even if one has backticks and the other doesn’t. In general, I think the usual compilation doesn’t care about backticks, it’s only us who care.

Reply:
That's true. Just seems so wrong that we have to extract text here.